eng/Build.props

```diff
@@ -3,8 +3,8 @@
     <ProjectToBuild Include="$(RepoRoot)src\**\*.csproj" Exclude="$(RepoRoot)src\Aspire.ProjectTemplates\templates\**\*.csproj" />
     <ProjectToBuild Include="$(RepoRoot)eng\dcppack\**\*.csproj" />
     <ProjectToBuild Include="$(RepoRoot)eng\dashboardpack\**\*.csproj" />
-    <ProjectToBuild Include="$(RepoRoot)playground\**\*.csproj" />
-    <ProjectToBuild Include="$(RepoRoot)tests\**\*.csproj"  />
+    <!-- <ProjectToBuild Include="$(RepoRoot)playground\**\*.csproj" />
```

Comment:
Playground apps and tests were disabled to allow the repo to build. I will continue iterating on this until ready and will remove this change as well as mark the PR as ready for review.